nondet/src/lib.rs

```diff
@@ -60,8 +63,26 @@ macro_rules! nondet_impl {
     };
 }
 
-pub(crate) use nondet_impl;
-use super::cvt;
+use cvt_core as cvt;
```

Comment:
these Solana related implementations must be factored out into a separate crate, like `solana-nd`

Reply:
we would need to define the `nondet` crate again in this case